### PR TITLE
[1.48.0] Image garbage collection

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: content/kotsadm/registries/kurl-registry.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+kubectl patch configmaps kotsadm-confg --type merge -p "{\"data\":{\"enable-image-deletion\":\"false\"}}"
+```
+
+Garbage collection is triggered automatically when a new application version is deployed.
+The [admin-console garbage-collect-images](/kots-cli/admin-console/garbage-collect-images/) command can be used to trigger it manually.
+
+### Restoring deleted images
+Deleted images may be reloaded from application and Admin Console [airgap bundles](/kotsadm/installing/airgap-packages/) using the [admin-console push-images](/kots-cli/admin-console/push-images/) command.
+See [installation instructions](/kotsadm/installing/airgap-packages/#kots-install) for an example of using this command.
+Registry address and namespace can be found on the Registry Settings page in Admin Console.
```
Comment (JustinN-Replicated) [inline content/kotsadm/registries/kurl-registry.md:37]: I noticed that there isn't a link to another portion of the docs here; having said that, I looked through and there isn't a good doc covering this tab of the admin console. Should I create a ticket & PR for this or do we think it is unneeded? I could likely write this up.

**Review by JustinN-Replicated — CHANGES_REQUESTED** [review]
Dmitriy - would like you to review the minor changes I made; if you think they are okay then I am okay to merge.

File: content/kotsadm/registries/kurl-registry.md (hunk 1 above)
Comment (divolgin) [reply content/kotsadm/registries/kurl-registry.md:37]: Yeah, the docs are definitely missing that.

**Comment by divolgin** [comment]
@JustinN-Replicated that all looks fine to me.

**Pull request merged**